Gemfile.lock

```diff
@@ -120,11 +127,11 @@ GEM
       simplecov (<= 0.13)
     coderay (1.1.1)
     concurrent-ruby (1.0.5)
-    coveralls (0.8.21)
+    coveralls (0.8.19)
```

Comment:
Some of your gems are older.  Are there particular reasons?  Gems where yours are older:
- `coveralls` 
- `simplecov`
- `thor`
(I think those are all of them)